src/connection/BaseConnectionOptions.ts

```diff
@@ -119,7 +119,15 @@ export interface BaseConnectionOptions {
         /**
          * Used to provide redis connection options.
          */
-        readonly options?: any;
+        readonly options?: any | {
```

Comment:
`options` are just some options to be sent to underlying caching driver, you shouldn't use it for your new option. Move tableName one level up.

Reply:
ok noted, thanks